test/test_artifacts/v1/amazon-codewhisperer-jupyterlab-ext.test.Dockerfile

```diff
@@ -0,0 +1,6 @@
+ARG COSMOS_IMAGE
+FROM $COSMOS_IMAGE
+
+ARG MAMBA_DOCKERFILE_ACTIVATE=1
+
+CMD ["python", "-c", "import amazon_codewhisperer_jupyterlab_ext"]
```

Comment:
Can you append the new test file to the list of test files in the test harness? https://github.com/aws/sagemaker-distribution/blob/main/test/test_dockerfile_based_harness.py

Ensure both CPU and GPU lists are updated. Thanks

Reply:
> Can you append the new test file to the list of test files in the test harness? https://github.com/aws/sagemaker-distribution/blob/main/test/test_dockerfile_based_harness.py
> 
> Ensure both CPU and GPU lists are updated. Thanks

Added, running the test now. It is taking long.